### DISCUSSION TITLE
Relay Modern Example doesn't work with Pagination

### DISCUSSION THREAD

**Discussion by octodhruv** [vercel/next.js#22617] [Help]
**What version of Next.js are you using?**

9.2.0

**What version of Node.js are you using?**

12.2.0

**What browser are you using?**

Chrome

**What operating system are you using?**

MacOS

**How are you deploying your application?**

CircleCI

**Describe the Bug**

Hi,

After following a set up detailed here to make relay modern work with next.js, I tried to set up a pagination container [1] following the relay docs, but it doesn't work as expected.

When I first attempt to load more data, nothing happens, and then from the second attempt it returns with an `invalid cursor` message.

Has anyone managed to successfully set up a pagination container using the documented example?

Thanks

--

[0] https://github.com/vercel/next.js/tree/canary/examples/with-relay-modern
[1] https://relay.dev/docs/en/pagination-container

**Expected Behavior**

Pagination container should work using the documented example.

**To Reproduce**

- Clone the `with-relay-modern` example
- attempt to implement a pagination container
- pagination fails

**Comment by leerob** [comment]
Converted this to a discussion. The example isn't broken, but you're looking for help using Relay + Next.js + Pagination 👍
